nd4j/RaspberryPi.md

```diff
@@ -0,0 +1,46 @@
+Please follow following instructions to build nd4j for raspberry PI:
+
+1. download cross compilation tools for Raspberry PI
+
+    ```
+    $ apt-get/yum install git rsync cmake
+    $ mkdir $HOME/raspberrypi
+    $ cd $HOME/raspberrypi
+    $ git clone git://github.com/raspberrypi/tools.git
+    $ export PATH=$PATH:$HOME/raspberrypi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin
+    $ mkdir $HOME/raspberrypi/rootfs
+    (Replace the ip address in the following command with the IP of a raspberry pi device)
+    $ rsync -rl --delete-after --safe-links pi@(192.168.1.PI):/{lib,usr} $HOME/raspberrypi/rootfs
```

Comment:
I'm pretty sure https://github.com/raspberrypi/userland provides the same thing...

Reply:
I'm not sure they are.  When I have a bit of time, I can try to audit this step to see what exactly is copied, but this took the better part of 30 minutes to complete, so I'm not sure I'd advise trying to recompile all the libraries and /bin files for a pi from scratch.

Reply:
I don't think we need all of the distribution! What do we need exactly that's not in tools and userland? Maybe we should fix that first.

Reply:
While true, this is taken from the general instructions for how to cross-compile for a raspi.
https://stackoverflow.com/questions/19162072/installing-raspberry-pi-cross-compiler
(See the accepted answer on this question)

Not being a dev on the deeplearning4j project, asking me to customize this is a bit of a lift. It's not exactly in my domain of expertise to know what dl4j needs on this level, so I would appreciate it if someone on the team could take a look at this, otherwise it's unlikely to be completed to a satisfactory level, especially considering things like GPU support. (which currently don't work with these instructions)

Reply:
Ah, I see.. Those instructions are not very good, hum.

Reply:
What was the issue with the original way of doing it without having to download anything from a device?

Reply:
I'll squash the rootfs folder and see if it compiles that way, TBH, I haven't tried, this was simply reorganizing the procedure previously listed in the RaspberryPi2.md file.

Reply:
Hallelulia, it compiles.  Squashed that part of the instructions and rpi.cmake file.